twitter_api/main.py

```diff
@@ -16,6 +21,158 @@ def before_request():
 
 
 # implement your views here
+def query_db(query, args=(), one=False):
+    cur = g.db.execute(query, args)
```

Comment:
imho better to use `cursor` than `cur` because you never know in the future what other attributes you have that may be called `cur`. `cursor` is very easily understood.